### PR TITLE
Variation of existing tool to delete all Bugs for a given source

### PR DESCRIPTION
**Pull request by andrewpollock**
For CVE unlaunching purposes.

Also update existing scripts to filter in non-warning inducing ways

### CODE REVIEW

File: tools/datafix/delete_bugs_with_source.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      "--project",
+      action="store",
+      dest="project",
+      default="oss-vdb-test",
+      help="GCP project to operate on")
+  args = parser.parse_args()
+
+  client = datastore.Client(project=args.project)
+
+  query = client.query(kind="Bug")
```
Comment (another-rex) [inline tools/datafix/delete_bugs_with_source.py:43]: This can run a lot faster as a keys_only query when verbose is set to false.

Comment (another-rex) [reply tools/datafix/delete_bugs_with_source.py:43]: But if it's already running at a decent speed no need to optimise it.

**Review by another-rex — APPROVED** [review]

File: tools/datafix/delete_bugs_with_source.py (hunk 1 above)
Comment (andrewpollock) [reply tools/datafix/delete_bugs_with_source.py:43]: If I'm reading https://cloud.google.com/datastore/docs/samples/datastore-keys-only-query#datastore_keys_only_query-python correctly, it seems like a pretty straightforward optimisation? Thank you for bringing this to my attention.

Comment (andrewpollock) [reply tools/datafix/delete_bugs_with_source.py:43]: Added in [16860f6](https://github.com/google/osv.dev/pull/1660/commits/16860f6df6bedd3b02ddf4cccb17aa0ec125e195)

File: tools/datafix/delete_bugs_with_source.py [hunk 2]
```diff
@@ -42,6 +42,7 @@ def main() -> None:
 
   query = client.query(kind="Bug")
   query.add_filter(filter=PropertyFilter("source", "=", args.source))
+  query.keys_only()
```
Comment (another-rex) [inline tools/datafix/delete_bugs_with_source.py:45]: Do you intend to print out the full bug when verbose is true? Or just the key, if you want to print out the full bug you probably want to only call this when verbose if false.

Comment (andrewpollock) [reply tools/datafix/delete_bugs_with_source.py:45]: Good point. Adjusted accordingly in 6631cd5

**Pull request merged**